deploy/resources/v0.8.0/addons/pipelinetemplates/s2i-go/s2i-go-build-deploy.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: s2i-go-build-and-deploy-pipeline
+  namespace: openshift
```

Comment:
@pradeepitm12 
is the `namespace: openshift` a hard constraint for this feature to work in ui.

if no, could you remove metadata.name from the pipeline samples.

if yes, this pr will need to handle that in the controller logic.

Reply:
Yes, for the time being, `namespace: openshift` is a hard constraint.